omnibus-nikos_arm64/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM arm64v8/ubuntu:16.04
```

Comment:
This is different from what we use for the system-probe `arm64` build. Do you foresee any problems caused by that?

Reply:
I don't believe so, just because that's also true for the `x64` build! Plus the `build_omnibus-nikos_arm64` job runs on the same gitlab runner as `build_system-probe-arm64`, so if the underlying architecture is the same then I think the important things like word size / endianness / etc. should be the same too, right?